tests/tests.rs

```diff
@@ -1171,9 +1127,13 @@ async fn s3_tests() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
     let host = std::env::var("SERVER_ENDPOINT")?;
     let access_key = std::env::var("ACCESS_KEY")?;
     let secret_key = std::env::var("SECRET_KEY")?;
-    let secure = std::env::var("ENABLE_HTTPS").is_ok();
+    let secure = !["no", "false", "0"]
+        .into_iter()
+        .any(|x| std::env::var("ENABLE_HTTPS").unwrap_or("no".into()) == x);
     let ssl_cert_file = std::env::var("SSL_CERT_FILE")?;
-    let ignore_cert_check = std::env::var("IGNORE_CERT_CHECK").is_ok();
+    let ignore_cert_check = !["no", "false", "0"]
+        .into_iter()
+        .any(|x| std::env::var("IGNORE_CERT_CHECK").unwrap_or("no".into()) == x);
```

Comment:
There are many options if you deal with on/off flag. Just leave it to current way.

Reply:
These env variables also compatible with our `mint` testing project. I would prefer to leave them as is.